deployments/helm/nvidia-dra-driver-gpu/templates/rbac.yaml

```diff
@@ -0,0 +1,81 @@
+{{- if .Values.serviceAccount.create -}}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "nvidia-dra-driver-gpu.name" . }}-clusterrole
+rules:
+- apiGroups: ["resource.nvidia.com"]
+  resources: ["computedomains"]
+  verbs: ["get", "list", "watch", "update"]
```

Comment:
Thanks for cleaning this up further. Question -- do we actually `update` computedomain objects themselves or just `computedomains/status`?